package.json

```diff
@@ -9,7 +9,7 @@
     "/schemas"
   ],
   "engines": {
-    "node": ">=20"
+    "node": ">=16"
```

Comment:
Is there something that needs node 16? Lets at least stick to versions that aren't EOL